resources/profiles/Zonestar.ini

```diff
@@ -383,289 +681,514 @@ machine_max_jerk_y = 10
-min_layer_height = 0.1
-max_layer_height = 0.3
-max_print_height = 400
-nozzle_diameter = 0.4
-retract_before_travel = 2
-retract_before_wipe = 0%
-retract_layer_change = 0
-retract_length = 6.5
-retract_length_toolchange = 0
-retract_lift = 0
-retract_lift_above = 0
-retract_lift_below = 0
-retract_restart_extra = 0
-retract_restart_extra_toolchange = 0
-retract_speed = 40
-start_gcode = M104 S[first_layer_temperature] ; set extruder temp\nM140 S[first_layer_bed_temperature] ; set bed temp\nM190 S[first_layer_bed_temperature] ; wait for bed temp\nM109 S[first_layer_temperature] ; wait for extruder temp\nG28 ; home all axes\nG1 Z5 F3000; lift nozzle\nG1 X5 Y5 F1500 ; move to prime\nG1 Z0.2 F3000 ; get ready to prime\nG92 E0 ; reset extrusion distance\nG1 Y200 E15 F600 ; prime nozzle\nG1 Y50 F5000 ; quick wipe\nG92 E0
-use_relative_e_distances = 0
+machine_max_acceleration_e = 10000
+machine_max_acceleration_extruding = 1500
+machine_max_acceleration_retracting = 1500
+machine_max_acceleration_travel = 1500
+machine_max_acceleration_x = 9000
+machine_max_acceleration_y = 9000
+machine_max_acceleration_z = 500
+machine_max_feedrate_e = 120
+machine_max_feedrate_x = 500
+machine_max_feedrate_y = 500
+machine_max_feedrate_z = 12
+machine_max_jerk_e = 2.5
+machine_max_jerk_x = 10
+machine_max_jerk_y = 10
+machine_max_jerk_z = 0.2
+machine_min_extruding_rate = 0
+machine_min_travel_rate = 0
 
-#Z8 with different types of hotend
-[printer:Zonestar Z8+M3 HOTEND]
+## common for Z5X
+[printer:*Z5X*]
 inherits = *common*
-printer_model = Z8
-printer_variant = 0.4
+printer_model = Z5X
 bed_shape = 0x0,310x0,310x310,0x310
-default_filament_profile = Generic PLA @Zonestar
-default_print_profile = 0.20mm @Zonestar MULTI
-deretract_speed = 30,30,30
-before_layer_gcode = G92 E0
-end_gcode = G28 X\nM106 S0 ; turn off FAN\nM104 S0 ; turn off extruder\nM140 S0 ; turn off bed\nM84 ; disable motors
-extruder_colour = #FF0000;#00FF00;#0000FF
-extruder_offset = 0x0,0x0,0x0
-max_layer_height = 0.3,0.3,0.3
 max_print_height = 400
-min_layer_height = 0.15,0.15,0.15
-nozzle_diameter = 0.4,0.4,0.4
-retract_before_travel = 2,2,2
-retract_before_wipe = 0%,0%,0%
-retract_layer_change = 0,0,0
-retract_length = 10,10,10
-retract_length_toolchange = 0,0,0
-retract_lift = 0,0,0
-retract_lift_above = 0,0,0
-retract_lift_below = 0,0,0
-retract_restart_extra = 0,0,0
-retract_restart_extra_toolchange = 0,0,0
-retract_speed = 35,35,35
-start_gcode = M104 S[first_layer_temperature] ; set extruder temp\nM140 S[first_layer_bed_temperature] ; set bed temp\nM190 S[first_layer_bed_temperature] ; wait for bed temp\nM109 S[first_layer_temperature] ; wait for extruder temp\nG28 ; home all axes\nG1 Z5 F3000 ; lift nozzle\nG1 X5 Y10 F1500 ; move to prime\nG1 Z0.2 F3000 ; get ready to prime\nT[initial_tool]\nG92 E0 ; reset extrusion distance\nG1 Y200 E15 F600 ; prime nozzle\nG1 Y100 F5000 ; quick wipe\nG92 E0
-use_relative_e_distances = 1
-wipe = 0,0,0
-layer_gcode = G92 E0
+travel_speed = 120
```

Comment:
This key does not belong to printer section. Remove it from all printer sections.

You can run prusaslicer with --loglevel=5 parameter to see all errors.